src/main/java/study/step4/Track.java

```diff
@@ -0,0 +1,47 @@
+package study.step4;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Track {
+    private static final String DELIMITER = ",|:";
+
+    private int attemptCount;
+    private Cars cars;
+
+    public Track(String carNames, int finish) {
+        this.attemptCount = finish;
+        createCars(carNames);
+    }
+
+    private void createCars(String carNames) {
+        List<Car> cars = new ArrayList<>();
+        for (String carName : carNames.split(DELIMITER)) {
+            cars.add(new Car(carName));
+        }
+        this.cars = new Cars(cars);
+    }
+
+    public void startRacing() {
+        cars.startRacing();
+        attemptCount--;
+    }
+
+    public Boolean isRaceEnd() {
```

Comment:
primitive 타입으로 반환해도 충분할 것 같습니다.
```suggestion
    public boolean isRaceEnd() {
```